src/index.js

```diff
@@ -318,6 +319,10 @@ function Postgres(a, b) {
     return new Parameter(x, 3802)
   }
 
+  function join(sep, xs) {
+    return xs.flatMap((x, i) => i ? Array.isArray(x) ? [sep, ...x] : [sep, x])
```

Comment:
Is this missing the second ternery response?  right now it reads as you get either get the spread, or non spread version with the separator applied, but in the case of the "first" value, we would normally just want x by itself right?

```js
return xs.flatMap((x, i) => i ? (Array.isArray(x) ? [sep, ...x] : [sep, x]) : x )
```
added parathesis to show groupings, they're not needed for the expression to evaluate correctly.

Reply:
oof
i really wish i could get these tests running